dll/win32/shimgvw/shimgvw.c

```diff
@@ -959,6 +975,55 @@ ImageView_OnSize(HWND hwnd, UINT state, INT cx, INT cy)
     }
 }
 
+static LRESULT
+ImageView_Delete(HWND hwnd)
+{
+    DPRINT1("ImageView_Delete: unimplemented.\n");
+    return 0;
+}
+
+static LRESULT
+ImageView_Modify(HWND hwnd)
+{
+    int nChars = GetFullPathNameW(currentFile->FileName, 0, NULL, NULL);
```

Comment:
Maybe it's cleaner to make the function call where you just below it check for failure, not many lines before.